### PR TITLE
Feature/overview

### PR DESCRIPTION
**Pull request by pushpanjalip**
### What this PR does

Fixes [AB#166](https://dev.azure.com/nlesc-team-cat/9cb0bc0a-da81-42e9-b2bf-7ada019f9189/_workitems/edit/166)

I have created components to show sector specific tabs and information.
Now we can see the sector specific information when clicked on sector tab

### CODE REVIEW

File: lib/ReactViews/SidePanel/SectorInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <span className={Styles.sectorTitle}>{sector.title}</span>
+          </div>
+          <div className={Styles.sectorInfo}>
+            <p>{sector.info}</p>
+          </div>
+        </>
+      );
+    }
+    return null;
+    // return (
```
Comment (ctwhome) [inline lib/ReactViews/SidePanel/SectorInfo.jsx:24]: is this something you want to keep as a reference?

File: lib/ReactViews/SidePanel/SectorTabs.jsx [hunk 2]
```diff
@@ -1,20 +1,68 @@
...
 import Icon from "../Icon";
 import Styles from "./sector_tabs.scss";
-const SectorTabs = createReactClass({
+
+class SectorTabs extends React.Component {
+  constructor(props) {
+    super(props);
+  }
   render() {
+    const sectors = [
```
Comment (ctwhome) [inline lib/ReactViews/SidePanel/SectorTabs.jsx:10]: good abstraction, later we could replace this array from the DB or external input, top!

**Review by ctwhome — APPROVED** [review]
is this feature branch finished or need more work? if it is finished I would say we can merge into receipt. thanks

**Comment by kretep** [comment]
Looks fine to me too. There seems to be a conflict though

**Review by kretep — APPROVED** [review]

**Pull request merged**